b2b-sample/libs/shared/util/util-front-end-util/src/lib/frontendUtil.tsx

```diff
@@ -0,0 +1,130 @@
+/**
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { SideNavItem } from "@b2bsample/shared/data-access/data-access-common-models-util";
+import AdminIcon from "@rsuite/icons/Admin";
+import DashboardIcon from "@rsuite/icons/legacy/Dashboard";
+import GearCircleIcon from "@rsuite/icons/legacy/GearCircle";
+import PageIcon from "@rsuite/icons/Page";
+import PeopleBranchIcon from "@rsuite/icons/PeopleBranch";
+import PhoneIcon from "@rsuite/icons/Phone";
+import SettingHorizontalIcon from "@rsuite/icons/SettingHorizontal";
+import ShieldIcon from "@rsuite/icons/Shield";
+import SpeakerIcon from "@rsuite/icons/Speaker";
+
+export const LOADING_DISPLAY_NONE = {
+    display: "none"
+};
+export const LOADING_DISPLAY_BLOCK = {
+    display: "block"
+};
+
+/**
+ * 
+ * @param scopes - scopes of the user
+ * @param itemScopes - scopes that required for this section
+ * 
+ * @returns `true` if user has the necessary scopes, else `false`
+ */
+function hideBasesdOnScopesSideNavItems(scopes: string[], itemScopes: string[]): boolean {
+    return itemScopes.every(scope => scopes.includes(scope));
+}
+
+/**
+ * hide content based on the user's realated privilages
```

Comment:
Start the sentence with a Capital letter and end with a full-stop.

Reply:
Fixed with https://github.com/wso2/samples-is/pull/416/commits/8b059f39358c5652ea99e473f4cf34e23ad95e0b